### PR TITLE
[TECH-88] add staff pictures

### PR DESCRIPTION
**Pull request by maxxfreund**

### CODE REVIEW

**Comment by maxxfreund** [comment]
Asking for review on this cause it's done for now, but we should really look into loading times for our images cause the preview build is superrrr slow 😢 ⌛

**Comment by peach3es** [comment]
each images profiles need to be max like 500-700kb, it's because they're like 2-5 mb at the moment

File: app/siteSettings/staff.json [hunk 1]
```diff
@@ -64,25 +64,25 @@
...
-      "img": "assets/staff/template.png"
+      "img": "assets/staff/jo.jpg"
     },
     {
       "name": "Alexander Smagorinski",
       "position": "Developer",
       "id": "12",
-      "img": "assets/staff/template.png"
+      "img": "assets/staff/pawelC-pink4.jpg"
     },
```
Comment (peach3es) [inline app/siteSettings/staff.json:74]: Not going to be part of the club any further so we can get rid of this

**Review by peach3es — COMMENTED** [review]
Looks good to me, finally have everyone's face on there. make sure to get rid of the section I reviewed

File: app/siteSettings/staff.json [hunk 2]
```diff
@@ -4,19 +4,19 @@
...
+      "img": "assets/staff/pawelC-pink1.jpg"
+    },
+    {
+      "name": "Aria Tessler",
+      "position": "VP Internal",
+      "id": "2",
+      "img": "assets/staff/aria.jpg"
     },
     {
       "name": "Peter Tian",
```
Comment (peach3es) [inline app/siteSettings/staff.json:22]: can you change the picture to the one I put for the social media post? plsssss

**Review by peach3es — CHANGES_REQUESTED** [review]
check the changes that I requested plss

**Pull request merged**